### PR TITLE
Download form summary

### PR DESCRIPTION
**Pull request by proteusvacuum**
This adds a link to download the form summary. Kriti wanted this to be able to do something with all required questions. 

@NoahCarnahan

### CODE REVIEW

File: corehq/apps/app_manager/views/app_summary.py [hunk 1]
```diff
@@ -264,6 +264,96 @@ def _get_form_actions(self, form):
...
+                    calculate=question_response.calculate,
+                    relevant=question_response.relevant,
+                    required="true" if question_response.required else "false",
+                )
+            )
+        return tuple(form_summary_rows)
+
+    def _get_form_sheet_name(self, module, form, language):
+        return "{} - {}".format(
+                _get_translated_module_name(self.app, module.unique_id, language),
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/app_summary.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/app_manager/views/app_summary.py [hunk 2]
```diff
@@ -264,6 +264,96 @@ def _get_form_actions(self, form):
...
+                _get_translated_form_name(self.app, form.get_unique_id(), language),
+            )
```
Comment (dimagimon) [inline corehq/apps/app_manager/views/app_summary.py]: E121 continuation line under-indented for hanging indent

**Review by NoahCarnahan — APPROVED** [review]

**Comment by czue** [comment]
(added "announce" label, fyi)

**Comment by czue** [comment]
(also this is great!)

**Pull request merged**